### DISCUSSION TITLE
Is it possible to access query params from OAuth Sign up from a DB function?

### DISCUSSION THREAD

**Discussion by Metaphysics0** [supabase/supabase#10533] [Questions]
When signing up with an OAuth provider, is it possible to access query params from a Database function, i.e. from the `raw_user_metadata` object that is included within the normal `auth.signUp()` method?

Thanks so much

**Comment by GaryAustin1** [comment]
You would have an insert trigger function on auth.users (like in the auth guide for setting up a user profile table).
https://supabase.com/docs/guides/auth/managing-user-data#using-triggers

Then you use
 `new.raw_user_meta_data ->> 'avatar_url'` or whatever you call your values.  They will be text type.  Or you can just store the entire json object in a column.
